packages/main/src/themes/TextArea.css

```diff
@@ -7,72 +7,112 @@
 :host {
 	width: 100%;
 	color: var(--sapField_TextColor);
+	min-height: var(--__ui5_textarea_min_height);
 	font-size: var(--sapFontSize);
 	font-family: "72override", var(--sapFontFamily);
 	font-style: normal;
-	border-color: var(--sapField_BorderColor);
-	border-radius: var(--sapField_BorderCornerRadius);
 	box-sizing: border-box;
-	line-height: 1.4;
-	letter-spacing: normal;
+	line-height: var(--_ui5_textarea_line_height);
+	Letter-spacing: normal;
 	word-spacing: normal;
 	margin: var(--_ui5_textarea_margin);
 }
 
-:host([disabled]) {
-	opacity: var(--_ui5_input_disabled_opacity);
-	cursor: default;
-	pointer-events: none;
-	border-color: var(--sapField_ReadOnly_BorderColor);
-	-webkit-text-fill-color: var(--sapContent_DisabledTextColor);
-	color: var(--sapContent_DisabledTextColor);
-}
-
-:host([disabled]) .ui5-textarea-inner {
+:host(:not([value-state]):not([readonly]):not([focused])) .ui5-textarea-wrapper {
 	background: var(--sapField_BackgroundStyle);
-	background-color: var(--_ui5_textarea_disabled_background_color);
-}
-
-:host([focused]) .ui5-textarea-inner {
-	outline: var(--_ui5_textarea_focus_outline);
-	outline-offset: var(--_ui5_textarea_focus_outline_offset);
+	background-color: var(--_ui5_textarea_background_color);
 }
 
-:host([focused]) .ui5-textarea-inner {
-	background-color: var(--sapField_Focus_Background);
-	background-image: none;
+:host(:not([value-state]):not([readonly]):not([disabled]):hover) .ui5-textarea-wrapper {
 	box-shadow: var(--_ui5_textarea_focus_box_shadow);
-}
-
-:host(:not([value-state]):not([readonly]):not([focused]):hover) .ui5-textarea-inner {
-	box-shadow: var(--_ui5_textarea_hover_box_shadow);
 	background: var(--sapField_Hover_BackgroundStyle);
 	background-color: var(--sapField_Hover_Background);
+	border: var(--_ui5_textarea_hover_border);
 }
 
-:host([value-state]:not([value-state="None"])[focused]) .ui5-textarea-inner {
-	outline: var(--_ui5_textarea_value_state_focus_outline);
-	outline-offset: var(--_ui5_textarea_value_state_focus_outline_offset);
+:host(:not([value-state]):not([readonly]):not([disabled]):not([focused]):hover) .ui5-textarea-wrapper {
+	box-shadow: var(--_ui5_textarea_hover_box_shadow);
 }
 
 .ui5-textarea-root {
-	height: 100%;
-	min-height: var(--_ui5_input_height);
+	width: 100%;
+	height: inherit;
+	min-height: var(--__ui5_textarea_min_height);
 	display: inline-flex;
 	vertical-align: top;
 	outline: none;
 	box-sizing: border-box;
 	border-radius: inherit;
 	border-color: inherit;
+	position: relative;
+	outline: none;
 }
 
-.ui5-textarea-inner {
+.ui5-textarea-wrapper {
+	position: relative;
+	box-sizing: border-box;
 	width: 100%;
+	border-bottom: var(--_ui5_textarea_wrapper_border);
```

Comment:
Why we use a parameter for border-bottom when we can just add "none" as a value

Reply:
Leftover from the POC I think. It was 2px transparent border initially. I removed the variable.